entries/PaTh.AWSLambda.Tracing.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<AspectLibrary>
+    <Name>PaTh.AWSLambda.Tracing (Dynatrace)</Name>
+    <Kind>Package</Kind>
+    <Categories>
+        <Category>Distributed Tracing</Category>
+        <Category>SDK</Category>
+    </Categories>
+    <Author>@dtpath</Author>
```

Comment:
I suggest to use your name Patrick Thurner as it makes it more relatable. Otherwise, use `dtPaTh` without the `@`.

Reply:
I changed it.